guides/connect-database.md

```diff
@@ -24,5 +24,5 @@ Depending on the dialect, you might need different options:
 
 - [Using SQLite](guides/using-sqlite.md)
 - [Using PostegreSQL](guides/using-postgresql.md)
-- [Using MySQL](guides/using-mysql.md)
+- [Using MySQL (MariaDB)](guides/using-mysql.md)
```

Comment:
I think MariaDB needs its own page. I will soon make an update to the docs so you can take that off and I'll take care of it no worries :)

Reply:
Okay! changes done! I think that is not neccesary to add a single section for MariaDB, literally use the same connection as MySQL. Maybe a little description/especification about MariaDB inside the same document, idk.

Reply:
Alright, I understand. It's just that there will be a little message saying MariaDB support comes from `deno_mysql` and is therefore using the MySQL connector right now.

I think MySQL users don't need to see this, and it is better to separate concerns.

Plus, if one day MariaDB gets its own connector, it will already have its own page to add more details